README.md

```diff
@@ -1,17 +1,100 @@
+for 592+ conda packages.
 
-    conda build <recipe directory>
+* Package maintainers SHOULD upgrade packages
+  when appropriate.
 
-You have to use conda >= 1.7
-[conda](https://github.com/continuumio/conda).
+* Conda recipes MAY run other scripts:
 
-See http://docs.continuum.io/conda/build.html for information on how to make a recipe,
-or just look at the examples here.
 
-This project (`conda-recipes`) is in the public domain.   Note that this statement
+  * at build time by specifying ``script:`` in ``meta.yaml``
+    (default: ``build.bat`` / ``build.sh``)
+  * at build time in `build.bat` (Windows) and `build.sh` (OSX, Linux)
+  * before the package is installed in ``pre-link.sh``
+  * after the package is installed in ``post-link.sh``
+  * before the package is removed in ``pre-unlink.sh``
+
+  http://conda.pydata.org/docs/build.html
+
+* Some of these recipes do intersect with
+  [Conda packages included with Anaconda](http://docs.continuum.io/anaconda/pkg-docs.html).
+
+* You can [search for Conda packages hosted in free public BinStar Channels](https://binstar.org/search?q=).
+
+
+## Requirements
+
+* [conda](https://github.com/conda/conda) >= 1.7
+  -- https://github.com/conda/conda#installation
+
+* [conda-build](https://github.com/conda/conda-build)
+  -- https://github.com/conda/conda-build#installation
+
+## Creating Conda Recipes
+See http://conda.pydata.org/docs/build.html for information
+on how to make a conda recipe, or just look at the [examples](#examples) here.
+
+    # Generate a conda recipe from PyPI (Python)
+    cd conda-recipes/
+    conda skeleton pypi arrow
+    conda skeleton pypi rdflib-jsonld
+
+    # Generate a conda recipe from CRAN (R)
+    cd conda-recipes/r-packages
+    conda skeleton cran r-chron
+    conda skeleton cran r-dplyr
+
+    # Generate a conda recipe from CPAN (Perl)
+    conda skeleton cpan DateTime
+    conda skeleton cpan RDF::Helper
+
+
+### Examples
+Conda recipes that build from git:
+
+* [conda/meta.yaml](https://github.com/conda/conda-recipes/blob/master/conda/meta.yaml)
+* [cookiecutter/meta.yaml](https://github.com/conda/conda-recipes/blob/master/cookiecutter/meta.yaml)
+* [rstudio/meta.yaml](https://github.com/conda/conda-recipes/blob/master/rstudio/meta.yaml)
+
+Conda recipes that build from PyPI, CRAN, CPAN:
+
+* [arrow/meta.yaml](https://github.com/conda/conda-recipes/blob/master/arrow/meta.yaml)
+* [r-packages/r-chron/meta.yaml](https://github.com/conda/conda-recipes/blob/master/r-packages/r-chron/meta.yaml)
+* [r-packages/r-dplyr/meta.yaml](https://github.com/conda/conda-recipes/blob/master/r-packages/r-dplyr/meta.yaml)
+* [perl/meta.yaml](https://github.com/conda/conda-recipes/blob/master/perl/meta.yaml)
+
+
+## Building Conda Recipes
+To build conda packages from a conda recipe:
+
+```bash
+git clone ssh://git@github.com/conda/conda-recipes
+recipe_name="conda"
+conda build "conda-recipes/${recipe_name}"
+ls -al ${CONDA_ROOT}/conda-bld/** | grep "${recipe_name}"
+```
```

Comment:
I added ``recipe_name`` as a path containing a '/' to demonstrate that conda-build accepts a path (there is no magic).

```bash
git clone ssh://git@github.com/conda/conda-recipes
conda build conda-recipes/conda
# ...
```

I added the CONDA_ROOT variable because different installations are different. (e.g. ``/opt/conda``, ``~/-wrk/-conda27`` (https://github.com/westurner/dotfiles/blob/release/0.8.3/etc/bash/08-bashrc.conda.sh).

I could list a number of CONDA_ROOT examples.

I think it's easier to understand that the prefix is CONDA_ROOT. I find this confusing:

    ~/{Anaconda,miniconda}/conda-bld/arch/